src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DecimalMathFuncs_NumberIsFloatDisabled_DVDecimal.txt

```diff
@@ -473,10 +473,10 @@ Error({Kind:ErrorKind.InvalidArgument})
 98765432198.12
 
 >> Round( 98765432198.1234567892, 100 )
-98765432198.1234567892
+Error({Kind:ErrorKind.Numeric})
 
 >> Round( 98765432198.1234567892, -100 )
-0
+Error({Kind:ErrorKind.Numeric})
```

Comment:
This doesn't have to be an error - if the second argument is negative, and (Abs(secondArg) + 1) > Log10(Abs(FirstArg)) then the result is 0